unidist/test/test_general.py

```diff
@@ -2,28 +2,44 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+import sys
 import pytest
 import time
 
 import unidist
 from unidist.config import Backend, CpuCount
+from unidist.core.base.common import BackendName
 from .utils import assert_equal, task, TestActor
 
 unidist.init()
 
 
-# FIXME: If we run all tests (i.e. pytest unidist/test), the tests are hanging.
-# However, if we remove third argument, the tests will be passed. This is true for MultiProcessing backend and
-# requires an investigation.
+@pytest.mark.skipif(
+    Backend.get() == BackendName.MP,
```

Comment:
see the comment above

Reply:
https://github.com/modin-project/unidist/pull/56#discussion_r766485355